cluster/gke/config-test.sh

```diff
@@ -18,11 +18,22 @@
 CLUSTER_NAME="${CLUSTER_NAME:-${USER}-gke-e2e}"
 NETWORK=${KUBE_GKE_NETWORK:-e2e}
 NODE_TAG="k8s-${CLUSTER_NAME}-node"
-IMAGE_TYPE="${KUBE_GKE_IMAGE_TYPE:-container_vm}"
+IMAGE_TYPE="${KUBE_GKE_IMAGE_TYPE:-cos}"
 ENABLE_KUBERNETES_ALPHA="${KUBE_GKE_ENABLE_KUBERNETES_ALPHA:-}"
 
 KUBE_DELETE_NETWORK=${KUBE_DELETE_NETWORK:-true}
 
+# Set e2e test context through NODE_OS_DISTRIBUTION.
+case $(kubectl get node -o=custom-columns=IMAGE:.status.nodeInfo.osImage | grep -v -m 1 IMAGE) in
+*"Container-Optimized OS from Google")
+	NODE_OS_DISTRIBUTION="gci"
+	;;
+*"Debian GNU/Linux 7 (wheezy)")
+	NODE_OS_DISTRIBUTION="debian"
+	;;
+esac
```

Comment:
I've set it to cos (which is gci) so that it at least mirrors the default state of a cluster.  It would be equally okay to remove the variable from this file.